NIFI-6755 "Chron Trigger Tutorial" link broken in NiFi User Guide (#3802)
apache/nifi: nifi-docs/src/main/asciidoc/user-guide.adoc

* The string `0 20 14 ? * MON-FRI` indicates that you want to schedule the processor to run at 2:20 PM every Monday through Friday.
* The string `0 15 10 ? * 6L 2011-2017` indicates that you want to schedule the processor to run at 10:15 AM, on the last Friday of every month, between 2011 and 2017.

> For additional information and examples on cron triggers, see tutorials in link:http://www.quartz-scheduler.org/documentation/[Quartz Documentation^].

Reviewer: I understand that the goal  of this fix is to replace the existing (broken) link with one that does not contain a specific version in the URL path and therefore is less likely to break again in the future.

However, the downside is that this change would require the reader of the NiFi documentation to both (1) know which version of Quartz is used by NiFi, and (2) navigate the Quartz Documentation to find the tutorial on cron triggers.

I suggest the following compromise, which includes the deep link to the specific tutorial at the correct version, and also a fallback link to the overall Quartz documentation:
 
```suggestion
For additional information and examples, see the link:https://www.quartz-scheduler.org/documentation/quartz-2.2.2/tutorials/tutorial-lesson-06.html[cron triggers tutorial] in the link:https://www.quartz-scheduler.org/documentation/[Quartz Documentation^].
```

Note: we prefer to use `https` links wherever possible, so I've updated that in my suggestion as well.

Let me know if you are ok with these suggestions.

Author: @kevdoran I have update the PR according to your suggestion. I think it is a good idea to give a version specific link (possible to break in the future) and a fallback general link.
